src/cachetools/__init__.py

```diff
@@ -595,21 +595,11 @@ def __getitem(self, key):
 )
 
 
-def cached(cache, key=keys.hashkey, lock=None, condition=None, info=False):
+def cached(cache, *, key=keys.hashkey, lock=None, condition=None, info=False):
     """Decorator to wrap a function with a memoizing callable that saves
     results in a cache.
 
     """
-    if isinstance(condition, bool):
-        from warnings import warn
-
-        warn(
-            "passing `info` as positional parameter is deprecated",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        info = condition
-        condition = None
```

Comment:
this is the typical type of bug that becomes impossible to make with keyword-only arguments